File: app/src/main/java/org/metabrainz/mobile/suggestion/SuggestionProvider.java
package org.metabrainz.mobile.suggestion;

import android.content.SearchRecentSuggestionsProvider;

public class SuggestionProvider extends SearchRecentSuggestionsProvider {

    public final static String AUTHORITY = "org.metabrainz.mobile.SearchSuggestionProvider";
    public final static int MODE = DATABASE_MODE_QUERIES;

    public SuggestionProvider() {
        setupSuggestions(AUTHORITY, MODE);
    }

}

File: app/src/main/java/org/metabrainz/android/presentation/features/suggestion/SuggestionProvider.kt
package org.metabrainz.android.presentation.features.suggestion

import android.content.SearchRecentSuggestionsProvider
import org.metabrainz.android.BuildConfig

class SuggestionProvider : SearchRecentSuggestionsProvider() {
    companion object {
        const val AUTHORITY = "${BuildConfig.APPLICATION_ID}.SearchSuggestionProvider"
        const val MODE = DATABASE_MODE_QUERIES
    }

    init {
        setupSuggestions(AUTHORITY, MODE)
    }
}